src/common/transformations/src/ov_ops/rms.cpp

```diff
@@ -29,11 +28,9 @@ void RMS::validate_and_infer_types() {
 
 std::shared_ptr<Node> RMS::clone_with_new_inputs(const ov::OutputVector& new_args) const {
     check_new_args_count(this, new_args);
-    return std::make_shared<RMS>(new_args.at(0),
-                                 new_args.at(1),
-                                 m_epsilon);
+    return std::make_shared<RMS>(new_args.at(0), new_args.at(1), m_epsilon);
```

Comment:
The `output_type` attribute should be provided to the cloned RMS op (but it was missed in the original file, so can be fixed separately):
```suggestion
    return std::make_shared<RMS>(new_args.at(0), new_args.at(1), m_epsilon, m_output_type);
```